server/lib/userlib.js

```diff
@@ -113,5 +114,55 @@ export default {
       console.error('Clearbit error', err);
       return cb(err);
     });
-  }
+  },
+
+  /*
+   * If there is a username suggested, we'll check that it's valid or increase it's count
+   * Otherwise, we'll suggest something.
+   */
+
+  checkOrSuggestUsername(user) {
```

Comment:
It's a weird method name. Check + Or don't go well together (what are you checking before getting to the OR part?). I'd prefer in this case simply createUsername()